presto-cassandra/src/test/java/com/facebook/presto/cassandra/CassandraServer.java

```diff
@@ -123,27 +114,19 @@ private static String prepareCassandraYaml()
         return yamlLocation.toAbsolutePath().toString();
     }
 
-    public static synchronized CassandraSession getSession()
+    public CassandraSession getSession()
```

Comment:
For line 109:

Since you removed data_directory from cu-cassandra.yaml, this is no longer needed. However we need to make sure the path you specified always exist and won't cause error on different platforms.

Reply:
Sure I have removed the line. Also the path is inside cassandra docker container and it will be independent of the platforms, so I believe this won't cause issues. I have updated the cherrypick id in the description of the PR.